src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    private static final Random random = new Random();
+    private final Color[] color = Color.values();
+
+    public static String getRandomColor() {
```

Comment:
The `getRandomColor()` method should not be static. You should be able to call this method on an instance of `ColorSupplier` class.